clang-format.py

```diff
@@ -33,11 +33,11 @@ def check_command_line_args(argv):
 
     if(len(argv) < len(required_args) + 1):
         for word in (['Usage:', argv[0]] + required_args):
-            print word,
-        print ''
+            print(word,)
+        print('')
         for word in (['E.g:', argv[0]] + example_args):
-            print word,
-        print ''
+            print(word,)
```

Comment:
This doesn't do anything in python 3. If the point is to print without a newline but a space after use
```
print(word, end='  ')
```